### ISSUE TITLE
FR: Export Slideshow as...

### ISSUE THREAD

**Issue by mmadson** [zsviczian/obsidian-excalidraw-plugin#1580]
After presenting a slideshow, it's common within an enterprise environment to send slides to attendees in addition to the recording. It'd be great if we can export the slideshow as perhaps a PowerPoint or even a multi-page PDF document or a series of images, something that can make it easier to deliver the presentation to attendees.

**Comment by sebastianpetrovski** [comment]
Agreed! As a freelance creative PDFs are used regularly to share information before, during and after a project is complete. I saw Excalidraw+ has the ability to export slide presentations as a PDF, having that feature inside of obsidian excalidraw plugin sounds dreamy.

**Comment by 4narchos** [comment]
Yesss this please.
As a medical doctor in residency the hability of having my motes both as lineal and mindmap in one place just improved my PKM a lot but the problem is I still have to make keynotes often, since slideshow I've got the chance of presenting my own notes so I don't have to work twice and the result is expectacular.
Problem is, often I have to share my slides and currently I can. 
It would just complete the circle for me and i'm sure for a lot of people.
